packages/back-end/src/util/integrations.ts

```diff
@@ -0,0 +1,102 @@
+import { DataSourceType } from "../../types/datasource";
+import {
+  InformationSchema,
+  RawInformationSchema,
+  Schema,
+  Table,
+} from "../types/Integration";
+
+type RowType = {
+  tableCatalog: string;
+  tableSchema?: string;
+  tableName?: string;
+  columnName?: string;
+};
+
+export function getPath(dataSource: DataSourceType, path: RowType): string {
```

Comment:
This probably doesn't need to be exported, right? It looks like it's only used in this file if I'm not mistaken.